docs/DEVELOPING.md

```diff
@@ -12,6 +12,143 @@
 - [PostgreSQL](https://www.elastic.co/guide/en/enterprise-search/master/connectors-postgresql.html)
 - [S3](https://www.elastic.co/guide/en/enterprise-search/master/connectors-s3.html)
 
+## Jira Connector
+
+The [Elastic Jira connector](../connectors/sources/jira.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+### Usage
+
+To use this connector as a **connector client**, use the **build a connector** workflow. See [Connector clients and frameworks](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+### Compatibility
+
+Jira versions 7 or later are compatible with Elastic connector frameworks.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Jira details](#gather-jira-details)
+2. [Configure Jira connector](#configure-jira-connector)
+
+#### Gather Jira details
+
+Collect the information that is required to connect to your Jira instance:
+
+- The domain where Jira is hosted.
+- Username for Jira server or account email for Jira cloud.
+- Password for Jira server or API Token for Jira cloud.
+- Project Keys to fetch the data from Jira server or cloud.
+- SSL certificate for a secure connection
+
+#### Configure Jira connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `data_source`
+
+Dropdown to determine Jira platform type. `Jira Cloud` if Jira cloud and `Jira Server` if Jira server. Default value is `Jira Cloud`.
+
+##### `username`
+
+The username of the account for Jira server.
+
+##### `password`
+
+The password of the account to be used for Jira server.
+
+##### `account_email`
+
+The account email for Jira cloud.
+
+##### `api_token`
+
+The API Token to authenticate with Jira cloud.
+
+##### `jira_url`
+
+The domain where Jira is hosted. Examples:
+
+  - `https://192.158.1.38:8080/`
+  - `https://test_user.atlassian.net/`
+
+##### `projects`
+
+Comma separated `Project Keys` to fetch the data from Jira server or cloud. Jira connector will fetch data from all projects present in the configured `project keys`, if the value is `*`. Default value is `*`. Examples:
```

Comment:
We have `projects`, `Project Keys`, and `project keys` here. Which one is correct?

Reply:
Providing a link to what project key means will be helpful: https://support.atlassian.com/jira-software-cloud/docs/what-is-an-issue/#Workingwithissues-Projectkeys

Reply:
@leemthompo `projects` is just a configuration parameter used in the connector code. User will see a label i.e. `Jira Project Keys` and will configure `project keys`. However, i'll provide a link for `project key` as @ppf2 suggested.

Reply:
OK but let's not confuse the reader by enclosing everything in backticks here.  :)